api/host.go

```diff
@@ -32,7 +32,7 @@ type (
 	// to /host/storage - a bunch of information about the status of storage
 	// management on the host.
 	StorageGET struct {
-		Folders []modules.StorageFolderMetadata `json:"folders"`
+		FolderMetadata []modules.StorageFolderMetadata `json:"foldermetadata"`
```

Comment:
yeah but I feel like 'Folders' is insufficiently clear. You aren't returning the actual folders, only information about the folders. You can't use the folders you receive to see the contents or make any changes.